pkgs/development/python-modules/portolan/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, isPy3k
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "portolan";
+  version = "1.0.1";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "fitnr";
+    repo = pname;
+    rev = "v"+version;
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Staged, will push as soon as everything is ready.